site/src/components/chart/tooltip.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: component
+title: Tooltip
+component: chart/tooltip.js
+namespace: chart
+
+example-code: |
```

Comment:
nah - it is a pretty boring example. Also, the way that playgrounds work at the moment (relying on the test fixture), it's not possible to add. Hopefully this will be resolved soon ...